doc/topics/storage.rst

```diff
@@ -311,3 +311,29 @@ the default storage class of the bucket, if the bucket is preexisting, or
 Sign blobs using IAM backed signing, rather than using GCP application credentials.
 The service account used needs to have ``iam.serviceAccounts.signBlob`` on the storage
 bucket.  This is available as part of ``roles/iam.serviceAccountTokenCreator``.
+
+Azure Blob Storage
+------------------
+This option will store your packages in a container in Azure Blob Storage.
+
+Set ``pypi.storage = azure-blob`` OR ``pypi.s3 = pypicloud.storage.AzureBlobStorage``
+
+A few key, required options are mentioned below.
+
+``storage.azure_storage_account_name``
```

Comment:
For consistency with the other storage options, you don't have to include the storage type in the option name. The storage type is specified in the `pypi.storage = azure-blob` argument, so all of the `storage.` arguments will apply to the azure blob backend